test/alternator/test_cql_rbac.py

```diff
@@ -669,6 +672,167 @@ def test_rbac_updatetable(dynamodb, cql):
                     authorized(lambda: tab.meta.client.update_table(TableName=tab.name,
                         BillingMode='PAY_PER_REQUEST'))
 
+# Above we checked UpdateTable's operation to change BillingMode, and that
+# it requires the ALTER permissions. But UpdateTable has a more interesting
+# ability - to *create* and to *delete* a GSI from the base table. In this
```

Comment:
So creating a table already with the GSI is not possible and we first need to create a table and then add a GSI to it via ALTER?

Reply:
No - we already supported since the first days of Alternator the `CreateTable` API to create a table with a GSI, what we're adding now is the `UpdateTable` API to modify an existing table by adding a GSI to it.
The `CreateTable` requires (unsurprisingly) the CREATE permissions, but also auto-grants all the permissions - including ALTER - to the role who was allowed to create the table, so this role is later able to add another GSI to this table if he wishes.